package.json

```diff
@@ -62,7 +62,7 @@
     "heroku-postbuild": "yarn run copy-polaris-tokens",
     "readme-update-version": "node ./scripts/readme-update-version",
     "version": "yarn run readme-update-version",
-    "storybook": "start-storybook -p 6006 --quiet",
+    "storybook": "start-storybook -p 6006 -c .storybook --quiet",
```

Comment:
Storybook looks in the `.storybook` folder by default so I don't think this config adds any value

Reply:
That's what I thought too but it doesn't seem to work without it 🤔